addons/account/models/chart_template.py

```diff
@@ -1219,6 +1225,10 @@ def _generate_tax(self, company, accounts_exist=False):
                     'cash_basis_transition_account_id': template.cash_basis_transition_account_id.id,
                 }
 
+                for existing_template, existing_tax in existing_template_to_tax:
+                    if template in existing_template.children_tax_ids and tax not in existing_tax:
```

Comment:
Shouldn't it rather be something like this ?

```suggestion
                    if template in existing_template.children_tax_ids and tax not in existing_tax.children_tax_ids:
```

Reply:
Of course indeed. Thanks !